tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/evaluation/ExpressionEvaluationNegativeTest.java

```diff
@@ -69,7 +69,18 @@ public void xmlTemplateEvaluationTest() throws BallerinaTestException {
     @Override
     @Test
     public void newConstructorEvaluationTest() throws BallerinaTestException {
-        // Todo
+        // Implicit new expressions
+        debugTestRunner.assertEvaluationError(context, "new", String.format(EvaluationExceptionKind
+                .CUSTOM_ERROR.getString(), "Implicit new expressions are not supported by the evaluator. Try using " +
+                "the equivalent explicit expression by specifying the class descriptor instead."));
+        debugTestRunner.assertEvaluationError(context, "new()", String.format(EvaluationExceptionKind
+                .CUSTOM_ERROR.getString(), "Implicit new expressions are not supported by the evaluator. Try using " +
```

Comment:
Can we make this error more understandable for new users may be with an example ie
```
Try using the equivalent explicit expression by specifying the class descriptor instead ie new T();
```

Reply:
Fixed with aec191d